lib/utils.js

```diff
@@ -0,0 +1,46 @@
+var Q = require('q');
+var es = require('.././config/es');
+
+var log = console.log;
+
+exports = module.exports = {
+  waitForElastic: function (waitCnt) {
+    var waitCntDef = 50;
+    waitCnt = waitCnt || 50;  // 40 requests by default
+
+    var deferred = Q.defer();
+
+    (function pinger(done) {
+      if(waitCnt != waitCntDef) log('requests left', waitCnt);
```

Comment:
I try to write code using [Google Javascript Styleguide](https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting). So, there is always space between if and bracket and always `{}` even for one line. 

```
if (...) {
    ...
} else {
    ...
}
```

Also, i have mistake with `is_elastic_enabled` variable, it should be named `isElasticEnabled`.